File: src/test/samples/validSamples/validSample.pl
#region FirstRegion
my $x = 42;
#endregion

#  region Second Region
sub my_function {
  #  region    InnerRegion    
  print "Hello, World!\n";
  #  endregion   ends InnerRegion  

  #region
  my $y = 24;
  #endregion
}
#  endregion

File: src/test/samples/validSamples/validSample.ml
(*#region FirstRegion*)
let x = 42
(*#endregion*)

(*  #region Second Region *)
let my_function () =
  (*  #region    InnerRegion   *)
  Printf.printf "Hello, World!\n"
  (*  #endregion   ends InnerRegion *)

  (* #region *)
  let y = 24 in ()
  (* #endregion *)
(*   #endregion *)
